lib/components/dropdown-item.vue

```diff
@@ -1,31 +1,21 @@
 <template>
     <b-link class="dropdown-item"
             role="menuitem"
```

Comment:
If they were bundled into the same component, which one would be the smartest default? I guess button, like you said, because you can detect link props to make it a b-link.